src/logical/permutator.rs

```diff
@@ -89,7 +89,7 @@ impl Permutator {
     /// Creates a [`Permutator`] based on one a list of [`VecT`].
     ///
     /// The sorting of values is done by taking the first [`VecT`] and sorting them according to these values.
-    /// If two elements are the equal to each other, the next [`VecT`] is taken to check if the comparison is different.
+    /// The sorting is done in a lexicographic order.
     /// If all [`VecT`] comparisons result in equality, the original order of the two values is preserved.
```

Comment:
Now you've replaced the middle step with a summary for the whole process, this doesn't make sense.

```markdown
Creates a [`Permutator`] from a list of [`VecT`], sorting in lexicographic order.

Returns [`Error::PermutationSortLen`]: Error#variant.PermutationSortLen if the given [`VecT`] differ in length.
```
is a perfectly valid doc for this method that has all the necessary information (in particular, which `Error` variant is returned), how sorting is done, and what the arguments are, yet does not contain unnecessary wording or details.